rosdep/base.yaml

```diff
@@ -1531,6 +1541,9 @@ libconfig-dev:
   gentoo: ['dev-libs/libconfig[cxx]']
   ubuntu: [libconfig-dev]
 libconsole-bridge-dev:
+  alpine:
+    source:
+      uri: 'https://github.com/at-wat/alpine-ros-sources/raw/master/console-bridge/console-bridge.rdmanifest'
```

Comment:
Related, it we can probably go ahead and merge this as is if it's working. But for greater maintainability and better use experience it would be great to submit this upstream for packing.